### ISSUE TITLE
OpenStack DMI detection fails on RedHat Openstack

### ISSUE THREAD

**Issue by quulah** [chef/ohai#1188]
## Description

Ohai is checking for the Manufacturer in DMI to [match](https://github.com/chef/ohai/blob/master/lib/ohai/plugins/openstack.rb#L32) OpenStack.

However, in this case dmidecode returns this:

```
System Information
        Manufacturer: Red Hat
        Product Name: OpenStack Compute
```

## Ohai Version

13.8.0

## Ohai Output

```
[2018-05-18T11:17:31+00:00] DEBUG: Plugin Openstack: openstack hint not present
[2018-05-18T11:17:31+00:00] DEBUG: Plugin Openstack: has_openstack_dmi? == false
[2018-05-18T11:17:31+00:00] DEBUG: Plugin Openstack: Node does not appear to be an Openstack node
[2018-05-18T11:17:31+00:00] DEBUG: Plugin Openstack took 0.0 seconds to run.
/opt/chef/embedded/lib/ruby/gems/2.4.0/gems/ohai-13.8.0/lib/ohai/system.rb:222:in `attributes_print': I cannot find an attribute named openstack! (ArgumentError)
```

**Comment by thommay** [comment]
can you run `ohai dmi/system` on one of these systems please? I've got a patch that ought to fix this but it'd be good to confirm the data.

**Comment by quulah** [comment]
Sure, I don't currently have access to the system, but I'll check it out as soon as I can.

**Comment by thommay** [comment]
@quulah the fix in #1192 should work for you.

**Comment by quulah** [comment]
Thanks @thommay. Yeah, looks good. We can't really jump to a newer version of Ohai for now, but are making do with a openstack.json hint.

There was actually a similar issue for Ansible, a while back, here's how they fixed it: https://github.com/ansible/ansible/commit/bda2f42f9772ab7176eb23c38935de5df3f38422

So looks like the product name was "OpenStack Nova" in earlier versions.
